capstone/src/main/resources/static/map.js

```diff
@@ -40,8 +39,28 @@ function createMarker(event,campusMap) {
   const newMarker = new google.maps.Marker({
     map: campusMap,
     title: event.eventTitle,
-    position: eventPosition
+    position: eventPosition,
   });
+  var eventInfoWindow = createInfoWindow(event, campusMap);
+  var eventContent = '<p id=mapContent>'+ event.eventTitle + '</p>';
+  newMarker.addListener('click', function() {
+    eventInfoWindow.setContent(eventContent);
```

Comment:
Would it work to just set the content in the `createInfoWindow()`, and only do the `open()` on the click event?